libs/wingc/src/jsify/snapshots/access_property_on_value_returned_from_collection.snap

```diff
@@ -78,7 +78,10 @@ class $Root extends $stdlib.std.Resource {
         super._registerOnLift(host, ops);
       }
     }
-    const s = ({"hello": "123"});
```

Comment:
Instead of this imperative code, can we use this syntax:

```js
const s = {
 ["expression" + "can" + "go" + "here"]: "123"
};
```

Reply:
OMG computed object properties, JS is just crazy. I'll fix.